ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/stun.j2

```diff
@@ -0,0 +1,40 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - stun #}
+{% if stun is arista.avd.defined %}
+
+## STUN
+{%     if stun.client is arista.avd.defined %}
+
+### STUN Client
+{%         if stun.client.server_profiles is arista.avd.defined %}
+
+#### Server Profiles
+
+| Server Profile | IP adress |
+| -------------- | --------- |
+{%             for server_profile in stun.client.server_profiles %}
```

Comment:
```suggestion
{%             for server_profile in stun.client.server_profiles | arista.avd.natural_sort('name') %}
```

Reply:
Fixed in https://github.com/aristanetworks/ansible-avd/pull/3147/commits/1e3ad5b31cf93920e70e558c79ca6aabea8566e2